pkg/release/release.go

```diff
@@ -217,3 +232,41 @@ func (r *ReleaseData) parse() error {
 		}
 	}
 }
+
+// SetReleaseDataForTests should only be called from tests. It sets the release information based on the supplied data.
+func SetReleaseDataForTests(data map[string][]byte) error {
```

Comment:
Are we able to place this in a _test file ?

Reply:
I can't 😢

Reply:
(files ending in _test are only compiled for tests in the package in question)